draft-davidben-tls-key-share-prediction.md

```diff
@@ -144,7 +144,7 @@ This section defines the `tls-supported-groups` SvcParamKey {{I-D.ietf-dnsop-svc
 
 ## Format
 
-The presentation `value` of the SvcParamValue is a comma-separated list ({{Appendix A.1 of I-D.ietf-dnsop-svcb-https}}) of decimal integers between 0 and 65536 in ASCII. Any other `value` is a syntax error. To enable simpler parsing, this SvcParam MUST NOT contain escape sequences.
+The presentation `value` of the SvcParamValue is a comma-separated list ({{Appendix A.1 of I-D.ietf-dnsop-svcb-https}}) of decimal integers between 0 and 65535 (inclusive) in ASCII. Any other `value` is a syntax error. To enable simpler parsing, this SvcParam MUST NOT contain escape sequences.
```

Comment:
I would also trim the last sentence.  Special parsing rules are special and therefore should be avoided.

Reply:
I don't feel very strongly either way. I guess it's a question of how parsers for these things tend to look. I just borrowed this phrasing (except then I picked up a typo) from https://www.rfc-editor.org/rfc/rfc9460.html#section-7.2-2, so I inferred banning escapes was preferable. 🤷 

Happy to do whatever DNS people like. I have no experience parsing these presentation values and likely never will.

(Figure I'll merge this PR as-is and we can sort that out separately.)